src/main/java/billennium/tests/mapper/DetailsMapper.java

```diff
@@ -10,7 +10,8 @@
 import java.util.List;
 
 @Component
-public class DetailsMapper {
+public class DetailsMapper {  //TODO uwaga do wszyskich Mapperow, mam wrazenie że niektore mappery mogą być
+    // uproszczone przy użyciu ModelMapper ze springa lub innej biblioteki.
```

Comment:
znam troche mapstruct ale zawsze wolałem ręcznie to pisać, jak znasz jakoś fajną bibliotekę to możemy wprowadzić

Reply:
ja używam ModelMappera przeważnie